### PR TITLE
Windows

### PR DESCRIPTION
**Pull request by shapi78**

### CODE REVIEW

File: Puppetfile [hunk 1]
```diff
@@ -28,7 +28,8 @@ mod 'puppetlabs/catalog_preview', :latest
 mod 'puppet/archive', :latest
 mod 'puppetlabs/inifile', :latest
 mod 'graduway/ec2tagfacts',
-  :git => 'https://github.com/graduway/ec2tagfacts'
+  :git => 'https://github.com/graduway/ec2tagfacts',
+  :branch => 'windows'
```
Comment (antgel) [inline Puppetfile]: This should be merged to master, please open a PR. Then we will submit a PR upstream. :)

File: hieradata/role/openvpn.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  net.ipv4.icmp_ignore_bogus_error_responses: 1
+  net.ipv4.conf.default.log_martians: 0
+
+openvpn::servers:
+  'graduway-vpn':
+    country: 'IL'
+    province: 'TLV'
+    city: 'Raanana'
+    organization: 'graduway.com'
+    email: 'it@graduway.com'
```
Comment (antgel) [inline hieradata/role/openvpn.yaml]: `development@`

File: hieradata/role/openvpn.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    server: '10.200.200.0 255.255.255.0'
+    push: ["route 10.10.0.0 255.255.0.0"]
+
+openvpn::client_defaults:
+  server: 'graduway-vpn'
+openvpn::clients:
+  'client1': {}
```
Comment (antgel) [inline hieradata/role/openvpn.yaml:35]: What are these 1/2/3? Can we have better identifiers?

File: hieradata/role/openvpn.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  net.ipv4.conf.default.send_redirects: 0
+  net.ipv4.icmp_ignore_bogus_error_responses: 1
+  net.ipv4.conf.default.log_martians: 0
+
+openvpn::servers:
+  'graduway-vpn':
+    country: 'IL'
+    province: 'TLV'
+    city: 'Raanana'
+    organization: 'graduway.com'
```
Comment (antgel) [inline hieradata/role/openvpn.yaml]: Graduway

File: hieradata/common.yaml [hunk 5]
```diff
@@ -19,7 +19,10 @@ psick::firstrun::linux_classes:
   hostname: '::psick::hostname'
   ec2tagfacts: '::ec2tagfacts'
 psick::firstrun::windows_classes:
-  hostname: '::psick::hostname'
+  aws-sdk: '::psick::aws::sdk'
+
+psick::firstrun::windows_reboot: false
+#  hostname: '::psick::hostname'
```
Comment (antgel) [inline hieradata/common.yaml]: Please remove commented code.

**Review by antgel — CHANGES_REQUESTED** [review]
Generally great, few small changes

**Review by antgel — APPROVED** [review]

**Comment by antgel** [comment]
@alvagante Hi, we forgot to get your review on this stuff. Please let us know what you think, thanks!

File: Puppetfile [hunk 6]
```diff
@@ -28,7 +28,7 @@ mod 'puppetlabs/catalog_preview', :latest
 mod 'puppet/archive', :latest
 mod 'puppetlabs/inifile', :latest
 mod 'graduway/ec2tagfacts',
-  :git => 'https://github.com/graduway/ec2tagfacts'
+  :git => 'https://github.com/graduway/ec2tagfacts',
```
Comment (alvagante) [inline Puppetfile:31]: We should start to specify versions for each module in Puppetfile, the default latest version is ok for demo purposes, but on real environments it's recommended to specify version names, in order not to have bad surprises when deploying modules in different times / places.
This comment applies not only to these new entries but to all the modules listed in the Puppetfile

**Review by alvagante — COMMENTED** [review]
Generatty this Looks ok to me

**Pull request merged**